README.md

```diff
@@ -119,6 +119,7 @@ If you're not inclined to make PRs you can tweet me at ```@ripienaar```
   * https://www.phplist.com/ - Hosted version allow 300 mails per month for free
   * https://www.mailjet.com/ - 6000 mails per month for free
   * https://www.sendinblue.com/ - 9000 mails per month for free
+  * https://www.campaignmonitor.com/ - 2500 mails to 500 subscribers per month for 9$
```

Comment:
I dont see how they have a usable free option, says you can us 'all the features free' till you have a campaign, this seems to be useless? 

Note though I've not used this software so might be missunderstanding the offering